specs/darwin/crashes.table

```diff
@@ -4,7 +4,7 @@ schema([
     Column("type", TEXT, "Type of crash log"),
     Column("pid", BIGINT, "Process (or thread) ID of the crashed process"),
     Column("path", TEXT, "Path to the crashed process"),
-    Column("crash_path", TEXT, "Location of log file"),
+    Column("crash_path", TEXT, "Location of log file", index=True),
```

Comment:
I don't see this in the c++ implementation, and looking at my machine, I think it's incorrect from a primary key perspective. 

`crash_path` could, maybe, be a primary key.